src/neuroglancer/datasource/dvid/frontend.ts

```diff
@@ -112,6 +110,7 @@ export class VolumeDataInstanceInfo extends DataInstanceInfo {
         'baseUrls': parameters.baseUrls,
         'nodeKey': parameters.nodeKey,
         'dataInstanceKey': dataInstanceKey,
+        'volumeType': this.volumeType,
```

Comment:
Determine the encoding here, e.g. based on the volumeType, and specify the encoding.